# Carregamento do dataset

### Informações gravadas em: https://github.com/bneurd/opt020

In [1]:
from scipy.io import loadmat

f = loadmat("../multi/Sub1_1_multitarget.mat")
print(f['Data'][0][0][8])
f['Data'][0][0][7].shape

[[7.5 8.2 7.  8.2 6.  7.5 6.  6.  8.2 8.2]]


(8192, 10)

Dado o exemplo acima, faça o carregamento dos cinco participantes, contatenando suas seções ou *trials*. Basta em um `for` externo aumentar a dimensionalidade para a quantidade de participantes e em um `for` interno, concatenar as seções com dez (10) *targets* cada uma.

Ao final, teremos um *shape* igual a `(5, 8192, 20)`.